packages/remix-express/server.ts

```diff
@@ -118,7 +118,7 @@ export async function sendRemixResponse(
   res.statusMessage = nodeResponse.statusText;
   res.status(nodeResponse.status);
 
-  for (let [key, values] of Object.entries(nodeResponse.headers.raw())) {
+  for (let [key, values] of nodeResponse.headers.entries()) {
     for (let value of values) {
       res.append(key, value);
     }
```

Comment:
This breaks for me since `entries` returns strings, not arrays in `values` like `raw` did so this then iterates over the characters in the `values` string.  

I think the original bug actually boils down to a bug in the node-fetch `entries` implementation (which we inherited into the Remix fork) - I pushed a potential fix here: https://github.com/remix-run/web-std-io/pull/39

If we were to merge that fix, then I think this would become:

```suggestion
  for (let [key, value] of nodeResponse.headers.entries()) {
    res.append(key, value);
```